src/transliteration/ui/WordInfo.tsx

```diff
@@ -59,7 +62,21 @@ export default function WordInfo({
   tokenClasses: readonly string[]
 }>): JSX.Element {
   const dictionary = useDictionary()
-
+  const { chapterId, lineNumber, variantNumber, textService } = useContext(
+    LineInfoContext
+  )
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const AlignedManuscriptLines = useMemo(
```

Comment:
Creates the list of manuscripts (for now, only shows the `sigla`). Unlike the `Score` component, which is within a `CollapsingRow` that gets created once and is mounted to the DOM, this part is re-rendered every time the popover is opened, incl. fetching the data.